fido2/client.py

```diff
@@ -262,29 +263,51 @@ def sign(self, app_id, challenge, registered_keys, timeout=None, on_keepalive=No
         }
 
 
-@unique
-class WEBAUTHN_TYPE(six.text_type, Enum):
-    MAKE_CREDENTIAL = "webauthn.create"
-    GET_ASSERTION = "webauthn.get"
+_CTAP1_INFO = b"\xa2\x01\x81\x66\x55\x32\x46\x5f\x56\x32\x03\x50" + b"\0" * 16
 
 
-_CTAP1_INFO = b"\xa2\x01\x81\x66\x55\x32\x46\x5f\x56\x32\x03\x50" + b"\0" * 16
+def running_windows_non_admin():
+    """
+    Determine if running on Windows with version >= 1903 
+    and not as an administrator, which would require the use of
+    the built in WebAuthN API in Windows.
+    """
```

Comment:
looks like they use `flake8` for linting which will complain about this docstring format. Are linting your addition on your end?

```suggestion
    """Determine if running on Windows with version >= 1903 and not admin.

    This would require the use of the built in WebAuthN API in Windows.

    """
```